### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by jha62** [yashmundra13/ORIE4741#15]
This project focuses on the effects of fracking. The goal is to predict if a public water source will be contaminated based on details about nearby fracking sites. The data includes details about each fracking site as well as details collected by the EPA about public water sources.

Likes:
1. The collection and cleaning of the dataset is well done and impressive. Preprocessing this data sounds like it was difficult.
2. Trying a simple linear model initially and recognizing it did not fit the data well was a good addition to the project. 
3. They tried a large number of different models and achieved impressive accuracy for the classifiers. These were all very interesting. 

Improvements:
1. I would have liked a short explanation of a Feed Forward Neural Net since it is not something we learned in class. A high level explanation of what it is and why it is useful would have been interesting.
2. The models had a very training accuracy compared to testing accuracy. A discussion about this possible overfitting could be included.
3. More visualizations would have been helpful to better understand the dataset. 

Overall I really enjoyed this project and the results were impressive.
